subprojects/plugins/src/test/groovy/org/gradle/api/tasks/testing/testng/TestNGOptionsTest.groovy

```diff
@@ -91,7 +92,7 @@ class TestNGOptionsTest extends Specification {
     }
 
     private TestNGOptions testNGOptionsWithPrefix(String prefix, boolean booleanValue, int intValue) {
-        return new TestNGOptions(layout)
+        return TestUtil.objectFactory().newInstance(TestNGOptions, layout)
```

Comment:
Would it make sense to add a shorthand for this via a static `TestUtil.newInstance()` method?

Reply:
I thought about it, but I wanted it to be clear in the diff that objectFactory was responsible for creating these instances

Reply:
If you want to reconsider, here's my vote for the shorter notation.